docs/site/examples.md

```diff
@@ -112,6 +112,24 @@ examples:
     links: 
     - language: powercli
       url: "/tree/master/examples/powercli/ha-restarted-vms"
+
+  - title: VMware Cloud on AWS SDDC Provisioned and Deletion Slack Notification
+    usecases:
+    - item: notification
+    id: vmware-cloud-ngw-slack
+    description: Send Slack message when a VMware Cloud on AWS SDDC is Provisioned or Deleted.
+    links:
+    - language: powershell
```

Comment:
`powershell` vs `powercli` - is that intentional?

Reply:
Yes. PowerShell is "minimum" environment and does not include the PowerCLI installation. It also makes it clear for non-PowerCLI use cases that automation can be developed w/o needing PowerCLI